bigframes/operations/_matplotlib/hist.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+
+import bigframes.constants as constants
+from bigframes.operations._matplotlib.core import MPLPlot
```

Comment:
import modules not methods/classes.

https://google.github.io/styleguide/pyguide.html#s2.2-imports